src/main/java/com/bzabek/ticketbookingapp/entities/Reservation.java

```diff
@@ -0,0 +1,146 @@
+package com.bzabek.ticketbookingapp.entities;
+
+import com.bzabek.ticketbookingapp.model.TicketType;
+
+import javax.persistence.*;
```

Comment:
taki importy utrudniają review bo nie zawsze wiadomo skąd jaka klasa pochodzi

Reply:
Rozumiem. Nie zwróciłem na to uwagi. Przetestowałem to i wygląda na to, że IDE automatycznie zastępuje kilka linijek z importami, tylko jedną linijką z gwiazdką*. W innych klasach do tego nie doszło ponieważ w klasie Reservation jest więcej importów z javax.persistance.